### PR TITLE
Basics notebooks

### PR DESCRIPTION
**Pull request by christian512**
Adds notebooks from the Basics Section on the QuTiP tutorial website.

### CODE REVIEW

**Review by hodgestar — COMMENTED** [review]
Questions!

- I'm not sure I like moving all of the images up to the top level. Ideally I would like images to be as close to their notebooks as possible. What was the reason for moving them?

- Why was the ultrastrong-coupling-groundstate move from v5 to v4? I would have expected it to appear in both.

**Comment by hodgestar** [comment]
I wonder if we shouldn't scrap the basics section entirely and move the two notebooks elsewhere. The superoperator tensor contracting is definitely not "basics", and neither is the ultrastrong coupling groundstate.

We can perhaps replace it with a paragraph on the index page which suggests some places to start.

**Comment by christian512** [comment]
- I wanted to move the images out of the `tutorials-vX` directories, so I don't have to add the images twice in the repository. I understand that having them close to the notebook helps with readability, but having the same images twice in the repository is also unnecessary.
- the `ultrastrong-coupling-groundstate` was just moved from `tutorials-vX` to `tutorials-vX/basics`.

**Comment by hodgestar** [comment]
I think we can live with having two copies of the images in order to have the tutorial-v4 and tutorial-v5 folders be a bit more independent.

As an example, in `try-qutip`, I copy only `tutorial-v4` and name it `tutorial` since only qutip 4 is installed, but with the images moved out this will now break. I could copy `images` too, but the point is that having the notebooks depend on files outside the directory subtree the notebook appears in is a bit messy for everyone trying to use a subset of the notebooks or even a single notebook.

Image that you want to download and use just one notebook. Previously you could download or copy the subfolder or just the notebook and the images and it worked. Now you have to download the whole repository structure, and you have to edit the notebook if you want to move it anywhere else.

File: tutorials-v4/lectures/Lecture-0-Introduction-to-QuTiP.md [hunk 1]
```diff
@@ -64,10 +64,10 @@ This will make the functions and classes in QuTiP available in the rest of the p
 
 At the heart of the QuTiP package is the `Qobj` class, which is used for representing quantum object such as states and operator. 
 
-The `Qobj` class contains all the information required to describe a quantum system, such as its matrix representation, composite structure and dimensionality. 
+The `Qobj` class contains all the information required to describe a quantum system, such as its matrix representation, composite structure and dimensionality.
 
 ```python
-Image(filename="images/qobj.png")
+Image(filename="../lectures/images/qobj.png")
```
Comment (hodgestar) [inline tutorials-v4/lectures/Lecture-0-Introduction-to-QuTiP.md:70]:
```suggestion
Image(filename="images/qobj.png")
```

**Review by hodgestar — COMMENTED** [review]
Looks good.

Just double-checking that `tutorials-v4/lectures/images/schematic-lasing-2ls.svg` is unused and was deleted intentionally?

**Comment by christian512** [comment]
@hodgestar, thanks for taking a look! 
The file is unused in the lectures.

**Review by hodgestar — APPROVED** [review]
Looks good to me. Happy for it to be merged once the CI runs pass.

**Pull request merged**